plone/schemaeditor/interfaces.py

```diff
@@ -157,6 +158,12 @@ def isValidFieldName(value):
 
 class INewField(Interface):
 
+    fieldset_id = Int(
+        title=_(u'Fieldset ID'),
+        description=_(u'Used to decide where to put this field.'),
+        required=True,
+    )
+
```

Comment:
How does this behave for already customized schematas? is there a migration step needed which sets the `fieldset_id` ?

Reply:
I don't have an example with already customized schematas. Maybe @mauritsvanrees could give this a try?

Reply:
@petschki My understanding is that the `INewField` interface is only used to get the fields for the "Add Field" form, and then what persists is the field itself (chosen with the `factory` field of this form). The `fieldset_id` is not passed in, so everything should continue to work just fine.